spec/views/users_show_spec.rb

```diff
@@ -0,0 +1,50 @@
+require 'rails_helper'
+
+RSpec.describe 'User show page - ', type: :feature do
```

Comment:
- [ ] Good job so far with your tests 👍🏽 . When you click on a user's post, kindly add a test that redirects to the post's show page as specified in the project requirement